modules/key-card/src/generateParamsForKeyCreation.ts

```diff
@@ -0,0 +1,43 @@
+import * as assert from 'assert';
+import { GenerateQrDataBaseParams, GenerateQrDataForKeychainParams, IDrawKeyCard } from './types';
+
+export function generateParamsForKeyCreation({
+  curve,
+  bitgoKeychain,
+  walletLabel,
+  keyCardImage,
+}: GenerateQrDataForKeychainParams & GenerateQrDataBaseParams): IDrawKeyCard {
+  assert(bitgoKeychain.commonKeychain, 'bitgoKeychain.commonKeychain is required');
+  return {
+    walletLabel,
+    keyCardImage,
+    curve,
+    qrData: {
+      user: {
+        title: 'A: Common Keychain',
+        data: bitgoKeychain.commonKeychain,
+        description: 'This is the common pub which is the equivalent of xpub (public key)\r\nof the key generated',
```

Comment:
Just checking, did product/technical-writers create these copies?

Reply:
Yes, Akshay did the copy and corrections.